csharp-ovh/Client/QueryStringParams.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Web;
+
+namespace Ovh.Api
+{
+    public class QueryStringParams : List<(string, string)>
+    {
+        private List<string> _existingKeys = new List<string>();
+
+        public void Add(string key, string value)
+        {
+            if(_existingKeys.Contains(key))
+            {
+                throw new ArgumentException("Duplicate keyword. This is currently not supported by OVH API");
+            }
+            Add((key, value));
+        }
+
+        public string ToQueryString(bool includeQuestionMark = true)
+        {
+            var sb = new StringBuilder();
+            if(includeQuestionMark)
+            {
+                sb.Append("?");
+            }
+
+            bool firstParam = true;
```

Comment:
I don't really get it here : why are you appending `&` inside the loop, instead of just doing it before?

Reply:
Will make more sense with the missing `firstParam = false;` line I think